IntegrationWithPharmacies/PharmacyRegistrationApi/Model/RegistrationInPharmacy.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace PharmacyRegistrationApi.Model
+{
+    public class RegistrationInPharmacy : Entity
+    {
+        public int PharmacyId { get; set; }
+        public String ApiKey { get; set; }
+        public String Name { get; set; }
+        public String Town { get; set; }
+
+
+        public RegistrationInPharmacy() : base() { }
+
+        public RegistrationInPharmacy(int id, int idPharmacy, String apiKeyPharmacy, String pharmacyName, String pharmacyTown) : base(id)
+        {
+            PharmacyId = idPharmacy;
+            ApiKey = apiKeyPharmacy;
+            Name = pharmacyName;
+            Town = pharmacyTown;
+        }
+        public RegistrationInPharmacy(int idPharmacy, String apiKeyPharmacy, String pharmacyName, String pharmacyTown)
+        {
+            PharmacyId = idPharmacy;
+            ApiKey = apiKeyPharmacy;
+            Name = pharmacyName;
+            Town = pharmacyTown;
```

Comment:
Change parameter name to match left side for example it should be "PharmacyId = pharmacyId"
Apply in other constructors.